ap-plus/config/_space.scss

```diff
@@ -1,4 +1,4 @@
 $space: 16px;
 
 // Bulma spacing
-$column-gap: $space / 2;
+$column-gap: $space * 0.5
```

Comment:
Using a divide is deprecated, so changed to multiply by 0.5 which returns the same result: 8px